seq-mainnet-1/RUN_VALIDATOR.md

```diff
@@ -0,0 +1,285 @@
+# Run Sequencer Validator
+
+- [Typical Setup](#typical-setup)
+- [Prerequisites](#prerequisites)
+- [Run an Ethereum Mainnet Node](#run-an-ethereum-mainnet-full-node)
+- [Further Sequencer Configurations](#further-sequencer-configurations)
+- [Run the Sidecar](#run-the-sidecar)
+- [Restart the Sequencer](#restart-the-sequencer)
+- [Creating an Account](#creating-an-account)
+- [Create the Validator](#create-the-validator)
+  - [What to Expect](#what-to-expect)
+  - [Tendermint KMS](#tendermint-kms)
+  - [Additional Advanced Configuration](#additional-advanced-configuration)
+
+## Typical Setup
+
+The validator setup will consist of a Fuel Sequencer, Sidecar, and a connection to an Ethereum Mainnet Node.
+
+![setup](Setup.drawio.png)
+
+## Prerequisites
+
+This guide assumes that you've gone through the steps to [run a Sequencer Node](./RUN_NODE.md).
```

Comment:
We should be careful as not all of the guide should be followed when running a validator. ex. setting the sidecar enabled to `false`. I think the best way forward is to have a single guide as I suggested above with the common parts extracted at the top, then the guide continues depending on whether the operator is setting up a full node or a validator node.